backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/export/svg/ImageRegistry.java

```diff
@@ -83,6 +90,16 @@ public StringBuilder getReferencedImageSymbols() {
         return symbols;
     }
 
+    private void logHeaders(HttpServletRequest request) {
+        Enumeration<String> headerNames = request.getHeaderNames();
+        if (headerNames != null) {
+            this.logger.info("The headers of the request are : " + headerNames.toString()); //$NON-NLS-1$
```

Comment:
for each header name, use `request.getHeaders(name)` to log their name and value, not just their name